tests/Tests/500__jupyterhub/test-jupyterhub-api-telemetry.robot

```diff
@@ -1,17 +1,21 @@
 *** Settings ***
 Resource    ../../Resources/ODS.robot
+Resource    ../../Resources/Common.robot
 Resource    ../../Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot
 Resource    ../../Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot
 Resource    ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
 Resource    ../../Resources/Page/OCPDashboard/OCPMenu.robot
 Library     SeleniumLibrary
 Test Setup    Test Suite For Web
 
+
 *** Test Cases ***
 Verify Telemetry Data Is Accessible
     [Tags]  Sanity
     ...     Tier1
     ...     ODS-347
+
+    Skip If RHODS Version Greater Or Equal Than    1.16.0
```

Comment:
will this be permanent? if so, then we could remove the TC. We have the branch for 1.15 which would run this one

Reply:
The idea is to remove the test in next sprint (or when possible)